frontend/packages/dev-console/src/components/pipelines/modals/common/PipelineParameterSection.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { FieldArray } from 'formik';
+import { TextInputTypes } from '@patternfly/react-core';
+import { InputField } from '@console/shared';
+import { PipelineParam } from '../../../../utils/pipeline-augment';
+import FormSection from '../../../import/section/FormSection';
+
+type ParametersSectionProps = {
+  parameters: PipelineParam[];
+};
+
+const PipelineParameterSection: React.FC<ParametersSectionProps> = ({ parameters }) => (
+  <FieldArray
```

Comment:
You can simply map over parameters array instead of using `FieldArray`. I think this was old implementation and you just moved it to a new file but would be a good candidate for refactoring.

Reply:
Hah, Divyanshi said the same thing when my PRs were stacking. Not my doing - 4.2 baggage. Will look a refactor ticket.

Reply:
https://issues.redhat.com/browse/ODC-3609